jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -330,6 +342,9 @@
                 export JENKINS_PUBLISHED_VERSION="ci/latest-1.2"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
                 export PROJECT="kubernetes-ingress-1-2"
+                # build-1.2 is not using the kubernetes-release-dev bucket yet
```

Comment:
@bprashanth did you intend to move the ingress jobs to 1.3? they're in the 1.3 section, but they're still testing 1.2.